### ISSUE TITLE
ServerInfo.properties updated in wrong location?

### ISSUE THREAD

**Issue by viper0131** [Unidata/tomcat-docker#53]
Hello,

The server version is still showing. 
When I look into the manipulated catalina.jar I see 2 ServerInfo.properties. The original in its original location and the adjusted one in the 'full path' location.

**Comment by viper0131** [comment]
this line:

    zip -ur ${CATALINA_HOME}/lib/catalina.jar \
${CATALINA_HOME}/lib/org/apache/catalina/util/ServerInfo.properties 

add's the full path

**Comment by julienchastang** [comment]
To elaborate `unzip catalina.jar`  yields:

```
./usr/local/tomcat/lib/org/apache/catalina/util/ServerInfo.properties
./org/apache/catalina/util/ServerInfo.properties
```

The first `ServerInfo.properties` is the correct one but in the wrong location. The second one remains unchanged so, yeah, this is a problem.

**Comment by julienchastang** [comment]
closed by #54
